web/controllers/task_controller.ex

```diff
@@ -40,12 +41,22 @@ defmodule CodeCorps.TaskController do
   def record(_conn, id), do: Task |> Repo.get(id)
 
   @spec handle_create(Plug.Conn.t, map) :: Ecto.Changeset.t
-  def handle_create(_conn, attributes) do
+  def handle_create(conn, attributes) do
+    project = Project |> Repo.get(attributes["project_id"])
+    if project.github_id do
+      current_user = Guardian.Plug.current_resource(conn)
+      github_id = github_module().create_issue(attributes, project, current_user)
+      attributes = Map.merge(attributes, %{"github_id" => github_id})
+    end
```

Comment:
There are no strong reasons right now, but

- It's more in line with how Phoenix 1.3 will structure modules.
- A clean, non-controller module will be easier to test for various outcomes.
- The code will very likely grow beyond these few lines.
- Generally, we like to keep our controller code as tiny as possible.